include/z64scene.h

```diff
@@ -58,103 +58,111 @@ typedef struct {
     /* 0x04 */ Vec3s* points; // Segment Address to the array of points
 } Path; // size = 0x8
 
-// Mesh headers
+// Room shapes
 
 typedef enum {
-    /* 0 */ MESH_HEADER_TYPE_0,
-    /* 1 */ MESH_HEADER_TYPE_1,
-    /* 2 */ MESH_HEADER_TYPE_2,
-    /* 3 */ MESH_HEADER_TYPE_MAX
-} MeshHeaderType;
+    /* 0 */ ROOM_SHAPE_TYPE_NORMAL,
+    /* 1 */ ROOM_SHAPE_TYPE_IMAGE,
+    /* 2 */ ROOM_SHAPE_TYPE_CULLABLE,
+    /* 3 */ ROOM_SHAPE_TYPE_MAX
+} RoomShapeType;
 
 typedef struct {
     /* 0x00 */ u8 type;
-} MeshHeaderBase; // size = 0x01
+} RoomShapeBase; // size = 0x01
 
 typedef struct {
     /* 0x00 */ Gfx* opa;
     /* 0x04 */ Gfx* xlu;
-} MeshHeader01Entry; // size = 0x08
+} RoomShapeDListsEntry; // size = 0x08
 
 typedef struct {
-    /* 0x00 */ MeshHeaderBase base;
+    /* 0x00 */ RoomShapeBase base;
     /* 0x01 */ u8 numEntries;
-    /* 0x04 */ MeshHeader01Entry* entries;
-    /* 0x08 */ MeshHeader01Entry* entriesEnd;
-} MeshHeader0; // size = 0x0C
+    /* 0x04 */ RoomShapeDListsEntry* entries;
+    /* 0x08 */ RoomShapeDListsEntry* entriesEnd;
+} RoomShapeNormal; // size = 0x0C
 
 typedef enum {
-    /* 1 */ MESH_HEADER1_FORMAT_SINGLE = 1,
-    /* 2 */ MESH_HEADER1_FORMAT_MULTI
-} MeshHeader1Format;
+    /* 1 */ ROOM_SHAPE_IMAGE_FORMAT_SINGLE = 1,
+    /* 2 */ ROOM_SHAPE_IMAGE_FORMAT_MULTI
+} RoomShapeImageFormat;
```

Comment:
i hope "image format" doesnt get confused with the usual meaning of image format, like for textures. but its prob fine

Reply:
oh I didn't think of that. "format" is just inherited from ancient docs so we could change it
maybe just "type" 8) ?

Reply:
-> `ROOM_SHAPE_IMAGE_AMOUNT_` :shrug: